src/styles/blocks/main.scss

```diff
@@ -0,0 +1,27 @@
+.catalog {
+  --column-count: 1;
+
+  padding: 50px 40px;
+  display: grid;
+  grid-template-columns: repeat(var(--column-count), 200px);
```

Comment:
The value `200px` for `grid-template-columns` is a magic number. It's recommended to use a variable for this value to increase code clarity and maintainability. Define it in `src/styles/utils/variables.scss` and use it here.